packages/IMAPClient-Core.package/ICCommandStorage.class/instance/add.for..st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+add: aCommand for: anAccount
+	
+	(self at: anAccount ifAbsent: [ self addAccount: anAccount ]) addFirst: aCommand
```

Comment:
This might cause the `ICEndPointCommandTest>>testFetchMailsBeginningAtWithFoundMails` test to fail on Ubuntu as `addAccount` doesn't return anything, so it implicitly returns the `ICCommandStorage` itself, which has no `addFirst` method.

Reply:
Good idea, we're going to look into that!